src/test/java/core/basesyntax/service/operation/ReturnOperationHandlerTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.operation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReturnOperationHandlerTest {
+    private static final Operation RETURN = Operation.RETURN;
+    private OperationHandler operationHandler;
+
+    @Before
+    public void setUp() {
+        operationHandler = new ReturnOperationHandler();
+    }
+
+    @Test
+    public void zeroPlusReturnFiveShouldEqualFive_Ok() {
+        FruitTransaction fruitTransaction = new FruitTransaction();
+        fruitTransaction.setName("orange");
+        fruitTransaction.setQuantity(5);
+        fruitTransaction.setOperation(RETURN);
+        operationHandler.handle(fruitTransaction);
+        Integer expectedQuantity = 5;
+        Integer actualQuantity = Storage.fruits.get("orange");
+        assertEquals("Test failed! Expected quantity should be " + expectedQuantity + " but it is "
+                + actualQuantity, expectedQuantity, actualQuantity);
+    }
+
+    @Test
+    public void fivePlusReturnSixShouldEqualEleven_Ok() {
```

Comment:
dont put numbers, in method. Methods like this can be named simply handle_validTransactionQuantity_Ok()